packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart

```diff
@@ -360,6 +366,9 @@ final AutoDisposeFutureProviderFamily<InstanceDetails, InstancePath>
       if (enumDetails != null) return enumDetails;
 
       final classInstance = await eval.getClass(instance.classRef, isAlive);
+      if (classInstance == null) {
```

Comment:
How is it possible that an object has no associated class?

Reply:
if isAlive got disposed part way through, eval will short circuit with a null response.

Reply:
Oh I see. I forgot that only "safeEval" throws if the isAlive was disposed.

Throwing "CancelledException" should be ideal then. That won't cause undesired logs as it is filtered

Reply:
Could you add a to-do that asks me to inspect all getClass/co calls to spot issues?

I doubt this is the only case.